Bio/Entrez/Parser.py

```diff
@@ -434,8 +434,6 @@ def startNamespaceDeclHandler(self, prefix, uri):
             # to find out their first and last invocation for each namespace.
             self.namespace_level[prefix] += 1
             self.namespace_prefix[uri] = prefix
-            assert uri == "http://www.w3.org/1998/Math/MathML"
```

Comment:
The uri is either "http://www.w3.org/1998/Math/MathML" or "http://www.w3.org/1999/xlink", isn't it? Then we can keep the assert if we allow both values.

Reply:
I think we can, actually my though was if URI is not from both the mentioned uri then also we should able to parse atleast some information.

Reply:
I think we can, actually my though was if URI is not from both the mentioned uri then also we should able to parse atleast some information.